public/examples/integrations/toolkits/code-sandbox/run_code_example_call_tool.js

```diff
@@ -0,0 +1,59 @@
+import { Arcade } from "@arcadeai/arcadejs";
+
+// Initialize the Arcade client
+const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable
+
+const USER_ID = "you@example.com";
+const TOOL_NAME = "CodeSandbox.RunCode";
+
+// Define the code to create a chart
```

Comment:
this comment looks wrong. 
```suggestion
// Define the code to merge sort a list
```

Looks like the comment is also wrong in Python

Reply:
Great catch